### ISSUE TITLE
Cannot connect error when logging in via corporate WIFI

### ISSUE THREAD

**Issue by afourney** [AzureAD/azure-activedirectory-library-for-cordova#105]
### Issue Description
This issue resembles the situation described here: [https://github.com/AzureAD/azure-activedirectory-library-for-cordova/issues/80](https://github.com/AzureAD/azure-activedirectory-library-for-cordova/issues/80)

When users attempt to login from a laptop connected to the corporate network, they receive the error:

"We can't connect to service right now. Check your internet connection or try this again later"

This occurs after they type their email address on the first screen, and are redirected.

### Repro Environment Details
1. We are are using version 8.1 of the plugin and are deploying to Windows 10 desktop UWP
2. The problem occurs only when users are connected to the corporate WiFi, or to public WiFi + VPN
3. I cannot provide any traces from Fiddler because of certificate pinning

**Comment by afourney** [comment]
Update: I think this issue it related to the ADAL bug here: [https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/222](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/222)

**Comment by afourney** [comment]
Ok, I appear to have fixed the problem by adding:
<Capability Name="privateNetworkClientServer" />
To the manifest file. 


Note that setting:
<preference name="adal-use-corporate-network" value="true" />
in cofig.xml *does not* resolve the issue. Instead, it results in the error "We received a bad request"

**Comment by nguyenjk** [comment]
I have same problem. @afourney were you able to resolve this issue?

**Comment by afourney** [comment]
@nguyenjk No. I was not able to resolve this problem with this plugin. I ended up implementing the OAuth exchange myself using the InAppBrowser plugin and MSAL. (https://docs.microsoft.com/en-us/azure/active-directory/active-directory-appmodel-v2-overview). Specifically:  https://docs.microsoft.com/en-us/azure/active-directory/active-directory-v2-protocols-oauth-code

We use the Secure Storage plugin (https://github.com/Crypho/cordova-plugin-secure-storage) to store then refresh tokens.

**Comment by JCpoz123** [comment]
Our app had this issue for a long time before we added the sharedUserCertificates and privateNetworkClientServer capabilities, AND upgrading to 3.19.6.  However, this seems to have been broken with 3.19.7.  The app no longers authenticates when connected to our VPN, and we are receiving the following error again:
"We can't connect to service right now. Check your internet connection or try this again later"

Reverting back to 3.19.6 corrects the issue.
